packages/contracts-bridge/contracts/test/utils/BridgeTest.sol

```diff
@@ -25,61 +27,86 @@ import {TypeCasts} from "@nomad-xyz/contracts-core/contracts/XAppConnectionManag
 
 import "forge-std/Test.sol";
 
-contract BridgeTestFixture is Test {
+contract BridgeTestFixture is NomadTest {
     using TypeCasts for bytes32;
     using TypeCasts for address;
     using TypeCasts for address payable;
 
-    // Local variables
-    uint256 mockUpdaterPK;
-    address mockUpdater;
+    // Fixtures
     address bridgeUser;
-    uint32 localDomain;
     uint256 bridgeUserTokenAmount;
+    ERC20Mock localToken;
 
-    // Remote variables
+    // Mock-only state
+    uint256 mockUpdaterPK;
+    address mockUpdater;
     bytes32 remoteBridgeRouter;
-    uint32 remoteDomain;
+    address mockReplica;
 
-    MockAccountant mockAccountant;
+    address home;
     MockHome mockHome;
-    address mockReplica;
+
+    // Real or mock
+    TokenRegistryHarness tokenRegistry;
     IBridgeRouterHarness bridgeRouter;
     XAppConnectionManager xAppConnectionManager;
     UpgradeBeaconController upgradeBeaconController;
     UpgradeBeacon tokenBeacon;
-
-    // Token Registry for all tokens in the domain
-    TokenRegistryHarness tokenRegistry;
     BridgeToken bridgeToken;
 
+    // eth-only
+    NFTRecoveryAccountantHarness accountant;
+
     // Implementation contract for all tokens in the domain
-    ERC20Mock localToken;
     address remoteTokenLocalAddress;
     bytes32 remoteTokenRemoteAddress;
     BridgeToken remoteToken;
 
-    function setUp() public virtual {
+    function setUp() public virtual override {
+        NomadTest.setUp();
+        setUp_mockState();
+        setUp_testFixtures();
+    }
+
+    function prankReplica(uint32 _d) internal {
+        vm.prank(xAppConnectionManager.domainToReplica(_d));
+    }
+
+    function prankReplica() internal {
+        prankReplica(remoteDomain);
+    }
+
+    function remoteRouter(uint32 _d) internal view returns (bytes32) {
```

Comment:
rename `remoteBridgeRouter`? governance also has remote routers

Reply:
it's a function on bridge test. no need